Tests/XcbeautifyLibTests/ParserTests.swift

```diff
@@ -175,4 +175,19 @@ final class ParserTests: XCTestCase {
         let captureGroup = try XCTUnwrap(parser.parse(line: input) as? SwiftTestingPassingArgumentCaptureGroup)
         XCTAssertEqual(captureGroup.numberOfArguments, 2)
     }
+
+    func testParsingLineSwiftTestingTestOutputWhenParallelTestingXCTestCases() throws {
+        let parser = Parser()
```

Comment:
`parser` is already defined by the test case, so you can delete this line and use that instance.